### PR TITLE
Fix all fbp query

### PR DESCRIPTION
**Pull request by pghorpade**
Connected to [APPS-1719](https://jira.library.ucla.edu/browse/APPS-1719), [APPS-1529](https://jira.library.ucla.edu/browse/APPS-1529)

This PR will add component library package to library website and fix GQL fragment queries and some miscellaneous tasks like fixing svg component path for impact etc.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -93,23 +48,15 @@ jobs:
     - run: npm run generate
       env:
         CRAFT_ENDPOINT: ${{ secrets.CRAFT_ENDPOINT }}
-        LIBCAL_ENDPOINT: ${{ secrets.LIBCAL_ENDPOINT }}
```
Comment (sourcefilter) [inline .github/workflows/ci.yml:96]: I think we need to keep LIBCAL_ENDPOINT? (Pointing to the proxy though - and we probably don't need to store it as a secret though that won't hurt anything)

File: .github/workflows/on-publish.yml [hunk 2]
```diff
@@ -21,11 +21,7 @@ jobs:
     - run: npm run generate
       env:
         CRAFT_ENDPOINT: ${{ secrets.CRAFT_PROD_ENDPOINT }}
-        LIBCAL_ENDPOINT: ${{ secrets.LIBCAL_ENDPOINT }}
```
Comment (sourcefilter) [inline .github/workflows/on-publish.yml:24]: (still need this? see ci.yml comment)

File: .github/workflows/ci.yml (hunk 1 above)
Comment (pghorpade) [reply .github/workflows/ci.yml:96]: Okay I will add this back

Comment (pghorpade) [reply .github/workflows/ci.yml:96]: I am doing this in nuxt config 
```
/*
   * Axios. Used for Libcal query.
   */
    axios: {
        baseURL:
      process.env.LIBCAL_ENDPOINT ||
      'https://test.proxy.calendar.library.ucla.edu/',
    },
```

**Review by sourcefilter — CHANGES_REQUESTED** [review]
Looks good! The only thing I noticed was missing info on Locations, which I assume was all coming from libcal and will be fixed once we point LIBCAL_ENDPOINT to the libcal proxy!

File: .github/workflows/on-publish.yml (hunk 2 above)
Comment (pghorpade) [reply .github/workflows/on-publish.yml:24]: It should be resolved now

**Review by sourcefilter — APPROVED** [review]
Libcal info is showing - though it turns out the ones I'd been checking before have the wrong slugs and aren't included in the production site, so I don't think we needed to add the LIBCAL_ENDPOINT environment variable back after all.

Anyway, 😎 🥳 👍

**Pull request merged**